op-service/txmgr/txmgr.go

```diff
@@ -562,38 +616,77 @@ func (m *SimpleTxManager) increaseGasPrice(ctx context.Context, tx *types.Transa
 		m.l.Info("re-estimated gas differs", "tx", tx.Hash(), "oldgas", tx.Gas(), "newgas", gas,
 			"gasFeeCap", bumpedFee, "gasTipCap", bumpedTip)
 	}
-	rawTx.Gas = gas
+
+	var newTx *types.Transaction
+	if tx.Type() == types.BlobTxType {
+		// Blob transactions have an additional blob gas price we must specify, so we must make sure it is
+		// getting bumped appropriately.
+		bumpedBlobFee := calcThresholdValue(tx.BlobGasFeeCap(), true)
+		if bumpedBlobFee.Cmp(blobBasefee) < 0 {
+			bumpedBlobFee = blobBasefee
+		}
+		maxBlobFee := new(big.Int).Mul(calcBlobFeeCap(blobBasefee), big.NewInt(int64(m.cfg.FeeLimitMultiplier)))
+		if bumpedBlobFee.Cmp(maxBlobFee) > 0 {
+			m.l.Warn("bumped blob fee getting capped at multiple of the suggested value", "bumped", bumpedBlobFee, "suggestion", maxBlobFee)
+			bumpedBlobFee.Set(maxBlobFee)
+		}
+		message := &types.BlobTx{
+			Nonce:      tx.Nonce(),
+			To:         *tx.To(),
+			Data:       tx.Data(),
+			Gas:        gas,
+			BlobHashes: tx.BlobHashes(),
+			Sidecar:    tx.BlobTxSidecar(),
+		}
+		if err := finishBlobTx(message, tx.ChainId(), bumpedTip, bumpedFee, bumpedBlobFee, tx.Value()); err != nil {
+			return nil, err
```

Comment:
Could we wrap this error with more context from this scope?

Reply:
done.

Reply:
> Could we wrap this error with more context from this scope?

No

- [ ] #10224